CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.21.0] - 2021-06-25
+### Added
+- Release the `conjur-config-cluster-prep` Helm chart. https://github.com/cyberark/conjur-authn-k8s-client#232
```

Comment:
I think for user impact for both of these by referencing the design doc for simplified dev experience.
The links should also be hyperlinks, not direct links.
Pretty sure you can combine this into one item, with 2 hyperlinks.

Something like this?:

```
### Added
- Introduces the `conjur-config-cluster-prep` and `conjur-config-namespace-prep` Helm charts.
  Together these charts simplify the deployment of Conjur-authenticated applications as part of
  the [Simplified Client Configuration](https://github.com/cyberark/conjur-authn-k8s-client/blob/master/design/simple-client-configuration.md) feature.
  [cyberark/conjur-authn-k8s-client#232](https://github.com/cyberark/conjur-authn-k8s-client/issues/232)
  [cyberark/conjur-authn-k8s-client#249](https://github.com/cyberark/conjur-authn-k8s-client/issues/249)
```

Reply:
Done